tests/fixtures/joint.py

```diff
@@ -63,6 +100,15 @@ async def joint_fixture(
             yield JointFixture(
                 config=config,
                 container=container,
-                mongodb=populated_mongodb_fixture,
+                mongodb=mongodb_fixture,
                 rest_client=rest_client,
             )
+
+
+def get_joint_fixture(scope: _ScopeName = "function"):
+    """Produce a joint fixture with desired scope"""
+    return pytest_asyncio.fixture(joint_fixture_function, scope=scope)
+
+
+mongodb_fixture = get_mongodb_fixture()
+joint_fixture = get_joint_fixture()
```

Comment:
```suggestion
mongodb_fixture = get_mongodb_fixture()
joint_fixture = get_joint_fixture()
```

I think we can/should removed these two lines sice we ar using the fixtures from conftest.py.

Reply:
Whoops. [adb86ee](https://github.com/ghga-de/mass/pull/9/commits/adb86ee9f1e04a2a8b94723143bdac5007b830bf)